.github/workflows/format_check.yml

```diff
@@ -12,59 +12,32 @@ jobs:
   format-check:
     # IMPORTANT: This name must match the require check name on the branch protection settings
     name: "Check for formatting errors"
-    runs-on: tooling-test-small
+    runs-on: ubuntu-latest
     steps:
-      - name: Checkout Airbyte
-        uses: actions/checkout@v3
+      - name: Checkout code
+        uses: actions/checkout@v4
         with:
           ref: ${{ github.head_ref }}
-          token: ${{ secrets.GH_PAT_APPROVINGTON_OCTAVIA }}
           fetch-depth: 1
-      - name: Run airbyte-ci format check [MASTER]
-        id: airbyte_ci_format_check_all_master
-        if: github.ref == 'refs/heads/master'
-        uses: ./.github/actions/run-airbyte-ci
-        continue-on-error: true
+      - name: Set up Python
+        uses: actions/setup-python@v5
         with:
-          context: "master"
-          dagger_cloud_token: ${{ secrets.DAGGER_CLOUD_TOKEN_2 }}
-          docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
-          docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
-          sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
-          github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
-          subcommand: "format check all"
+          python-version: "3.10"
 
-      - name: Run airbyte-ci format check [PULL REQUEST]
-        id: airbyte_ci_format_check_all_pr
-        if: github.event_name == 'pull_request'
-        uses: ./.github/actions/run-airbyte-ci
-        continue-on-error: false
-        with:
-          context: "pull_request"
-          dagger_cloud_token: ${{ secrets.DAGGER_CLOUD_TOKEN_2 }}
-          docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
-          docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
-          sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
-          github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
-          subcommand: "format check all"
+      - name: Install `airbyte-ci` CLI
+        # We use the stable `airbyte-ci` from `master`. Intentionally ignores any CI changes
+        # local to this branch.
+        run: >
+          pip install
+          git+https://github.com/airbytehq/airbyte.git#subdirectory=airbyte-ci/connectors/pipelines
```

Comment:
This would work pretty much in any repo if we copy and paste this action, right? We could do relative path here instead, but that would mean that it's less portable?

Reply:
Yes. This works in any repo - and seemed especially appropriate if there's a chance we're running in a fork. :)

Reply:
@natikgadzhi - I want to propose to you and @alafanechere that this be the default/recommended whenever (1) you don't trust the local branch, or (2) you don't have the `airbyte` repo checked out.